conflation/main.py

```diff
@@ -28,13 +28,27 @@ def main():
         help="The number of processes to use to make requests, by default your # of cpus",
         default=multiprocessing.cpu_count(),
     )
+    arg_parser.add_argument(
+        "--valhalla_url",
+        type=str,
+        help="Base URL for an active Valhalla service",
+        required=True,
+    )
+    arg_parser.add_argument(
+        "--valhalla_headers",
+        type=str,
+        help="Additional http headers to send with the requests. Follows the http header spec, eg. some-header-name: some-header-value",
+        action="append",
+        nargs="*",
+    )
```

Comment:
no need to do it now but i was expecting we would do the same thign we did for `--config` that is we provide a json blob that configures the mapmatching just like we provide a json blob that configures the trace data pulling.

Reply:
Oh I took it from your implementation in the PR you sent, I thought it was some sort of standard for passing in headers haha